src/components/shared/chart/Viewport.js

```diff
@@ -779,12 +781,12 @@ export const withChartViewport: WithChartViewport<*, *> =
         window.removeEventListener('resize', this._setSizeNextFrame, false);
         window.removeEventListener('mousemove', this._mouseMoveListener, true);
         window.removeEventListener('mouseup', this._mouseUpListener, true);
-        if (this._container) {
-          this._container.removeEventListener(
-            'wheel',
-            this._mouseWheelListener,
-            { passive: false }
-          );
+        const container = this._container;
+        if (container) {
+          getResizeObserverWrapper().subscribe(container, this._setSize);
```

Comment:
Shouldn't this be `unsubscribe` instead? :)

Reply:
that is... very likely :-) thanks for noticing!